### PR TITLE
Audio format cleanup

### PR DESCRIPTION
**Pull request by ranj063**

### CODE REVIEW

File: tools/topology/topology2/include/pipelines/cavs/deepbuffer-playback.conf [hunk 1]
```diff
@@ -47,28 +48,31 @@ Class.Pipeline."deepbuffer-playback" {
...
-				in_valid_bit_depth	32
-				out_bit_depth		32
-				out_valid_bit_depth	32
-			}
+			Object.Base.output_audio_format [
+				{
+					out_bit_depth		32
+					out_valid_bit_depth	32
+				}
+			]
```
Comment (plbossart) [inline tools/topology/topology2/include/pipelines/cavs/deepbuffer-playback.conf:75]: the comments on line 54 and 69 are no longer relevant, remove?

Comment (ranj063) [reply tools/topology/topology2/include/pipelines/cavs/deepbuffer-playback.conf]: this patch was from a different PR, will fix it in a follow up

**Review by plbossart — APPROVED** [review]

File: tools/topology/topology2/include/pipelines/cavs/gain-copier-capture.conf [hunk 2]
```diff
@@ -45,59 +46,89 @@ Class.Pipeline."gain-copier-capture" {
...
+
+			Object.Base.output_audio_format [
+				{
+					out_bit_depth		32
+					out_valid_bit_depth	32
+				}
+				{
+					out_bit_depth		16
+					out_valid_bit_depth	16
+				}
```
Comment (bardliao) [inline tools/topology/topology2/include/pipelines/cavs/gain-copier-capture.conf:102]: Does the order matter? The existing code format 1 is 16 bit and format 2 is 32 bit. But after the change output format 1 is 32 bit and output format 2 is 16 bit.

**Review by kv2019i — APPROVED** [review]

File: tools/topology/topology2/include/pipelines/cavs/gain-copier-capture.conf (hunk 2 above)
Comment (lgirdwood) [reply tools/topology/topology2/include/pipelines/cavs/gain-copier-capture.conf:102]: order in topology should not matter as FW should iterate over configs to match requested based on copier use case.

**Review by lgirdwood — APPROVED** [review]

File: tools/topology/topology2/include/pipelines/cavs/gain-copier-capture.conf (hunk 2 above)
Comment (ranj063) [reply tools/topology/topology2/include/pipelines/cavs/gain-copier-capture.conf:102]: yes, order doesnt matter

**Pull request merged**